central/risk/multipliers/deployment/service_account_permissions_test.go

```diff
@@ -214,7 +215,7 @@ func TestPermissionScore(t *testing.T) {
 
 	for _, c := range clusterCases {
 		t.Run(c.name, func(t *testing.T) {
-			ctx := context.Background()
+			ctx := sac.WithGlobalAccessScopeChecker(context.Background(), sac.AllowAllAccessScopeChecker())
```

Comment:
nit: `sac.WithAllAccess(context.Background())` does the same, and might be easier to understand for an external reader (also applies to other occurrences in the PR).

Reply:
➕